crates/project_core/Cargo.toml

```diff
@@ -0,0 +1,50 @@
+[package]
+name = "project_core"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+test-support = [
+    "client/test-support",
+    "language/test-support",
+    "settings/test-support",
+    "text/test-support",
+    "gpui/test-support",
+]
+
+[dependencies]
+anyhow.workspace = true
+client.workspace = true
+clock.workspace = true
+collections.workspace = true
+fs.workspace = true
+futures.workspace = true
+fuzzy.workspace = true
+git.workspace = true
+gpui.workspace = true
+ignore = "0.4.22"
```

Comment:
Can we hoist this up to a workspace-level dependency?

Reply:
Damn, it'd be nice to have `cargo add` do that automatically with something like `--workspace` argument.